FreeBSD make fail

When I attempt to install netdata on FreeBSD 11.1 p6, make fails with the following error:

--- apps.plugin ---
/tmp//ccptRm1K.ltrans0.ltrans.o: In function `main':
<artificial>:(.text.startup+0x2835): undefined reference to `arl_free'
collect2: error: ld returned 1 exit status
*** [apps.plugin] Error code 1

make[2]: stopped in /usr/home/denningsrogue/netdata/src

Any help would be appreciated. (I also tried to install from ports and pkgs.  The installs seems to finish without issue, but netdata is inaccessible even though the service is running.)

--- a/src/apps_plugin.c
+++ b/src/apps_plugin.c
@@ -292,7 +292,9 @@ struct pid_stat {
     kernel_uint_t status_rssfile;
     kernel_uint_t status_rssshmem;
     kernel_uint_t status_vmswap;
+#ifndef __FreeBSD__
     ARL_BASE *status_arl;
+#endif
 
     kernel_uint_t io_logical_bytes_read_raw;
     kernel_uint_t io_logical_bytes_written_raw;
@@ -696,7 +698,9 @@ static inline void del_pid_entry(pid_t pid) {
     freez(p->fds_dirname);
     freez(p->stat_filename);
     freez(p->status_filename);
+#ifndef __FreeBSD__
     arl_free(p->status_arl);
+#endif
     freez(p->io_filename);
     freez(p->cmdline_filename);
     freez(p->cmdline);

